Changes.md

```diff
@@ -4,7 +4,15 @@
 Improvements
 ------------
 
-- EditScope Menu : Changed the order of listed Edit Scopes to be alphabetical.
+- EditScope Menu : 
+  - Changed the order of listed Edit Scopes to be alphabetical.
+  - Added an icon to menu items that are ancestors of the active Edit Scope.
+  - Menu items that only have single item in their sub-menu are now collapsed into the parent item.
```

Comment:
have single item -> a single item

Reply:
Fixed in c978af97c9a018083c707a8d94bc7538c3a826a7